pkg/kapp/config/default.go

```diff
@@ -237,6 +230,10 @@ diffAgainstLastAppliedFieldExclusionRules:
 - path: [metadata, annotations, "deployment.kubernetes.io/revision"]
   resourceMatchers: *appsV1DeploymentWithRevAnnKey
 
+- path: [status]
```

Comment:
Not super clear what it will do, the copy part is already removed above then what is the purpose of adding it here?

Reply:
Umm, do you mean you are not clear about why I have used `diffAgainstLastAppliedFieldExclusionRules`?

Reply:
Initially was not clear but looks like this is for not to consider `status` field while diffing right?

Reply:
Yep, that's right.